torch/ao/quantization/quantizer/quantizer.py

```diff
@@ -157,6 +157,13 @@ class QuantizationAnnotation:
 
 
 class Quantizer(ABC):
+
+    # transform the model before annotation, to make sure we can annotate the model properly
+    # for example, we can decompose some compound operators here, or change scalar argument to Tensor
+    # so we can annotate the argument
```

Comment:
```suggestion
     # Allows for user defined transforms to run before annotating the graph. This allows quantizer to allow quantizing part of the model
     # that are otherwise not quantizable. For example quantizer can a) decompose a compound operator like scaled dot product attention, into bmm and softmax if quantizer knows how to quantize bmm/softmax but not sdpa
     # or b) transform scalars to tensor to allow quantizing scalares. 
```